src/components/LogPane/index.tsx

```diff
@@ -37,6 +38,15 @@ const LogPane: React.FC<LogPaneProps> = ({
     listRef.current?.recomputeRowHeights();
   }, [listRef, cache, wrap, matchingLines, expandableRows, prettyPrint]);
 
+  useEffect(() => {
```

Comment:
If we're going to run a `useEffect` on component mount, then I don't think `initialScrollIndex` needs to be a state in the parent component anymore.

Reply:
I replaced it with a useMemo